src/task/src/metrics/inputs.ts

```diff
@@ -22,6 +22,7 @@ export default class Inputs {
   private _alwaysCloseComment: boolean = false
   private _fileMatchingPatterns: string[] = []
   private _codeFileExtensions: Set<string> = new Set<string>()
+  private _changePrTitle: boolean = true
```

Comment:
Please ensure that when this is not specified in the YAML, it defaults to `false`. I understand that you've taken steps to account for the parameter not being set, but there is a possibility that it will default to false, so please verify this. The change needs to be designed in a way to prevent existing customers encountering behaviour changes.

Reply:
Unit tests have been added to cover this.  I have not run any manual tests.

Reply:
Thanks for adding unit tests. There are a lot of customers using this, and we cannot afford to break them, so this really needs manual testing. Is that something you can take on, or do you need us to work on it? If you need us to help, I'm not sure we'll be able to get to it next week and it might have to be the week afterwards – but rest assured that I'm still committed to getting your contribution merged.